warehouse/transformer/transformer.go

```diff
@@ -27,6 +27,14 @@ const (
 	redshiftStringLimit = 512
 )
 
+// Compile-time check to ensure ValidationError struct remains unchanged.
```

Comment:
Are there other structs for which we should add this check ?

Reply:
No. Only this is sufficient.